simple_examples.py

```diff
@@ -67,18 +67,28 @@ def get_line_distance_polytype(line: Union[Line, Tuple[Point, Point], Tuple[Tupl
     )
 
 
+# Reference material for enums - https://docs.python.org/3/library/enum.html#creating-an-enum
+# It is important to note that enums are functional constants. We should refra
+    # Note Nomenclature
+    # The class TextAlign is an enumeration (or enum)
+    # The attributes TextAlign.left, TextAlign.center, etc., are enumeration members (or enum members) and are functionally constants.
+    # The enum members have names and values (the name of TextAlign.left is left, the value of TextAlign.hidden is 0, etc.)
+
 @typechecked
 class TextAlign(Enum):
     left: str = 'left'
     right: str = 'right'
     center: str = 'center'
+    #used to highlight the difference between name and value in enums
+    hidden: int = 0
```

Comment:
Based on reading the documentation you linked on Enums, perhaps one of these are a better way to write this:
```python
TextAlign = Enum('TextAlign', 'LEFT RIGHT CENTER')
```
or
```python
TEXT_ALIGN_KEYS = ['LEFT', 'RIGHT', 'CENTER']
TextAlign = Enum('TextAlign', ' '.join(TEXT_ALIGN_KEYS))
```
or
```python
from enum import Enum, auto
class TextAlign(Enum):
    LEFT = auto()
    RIGHT = auto()
    CENTER = auto()
```

Reply:
I think that I prefer the verbose way you were doing it first, unless there are benefits in accessing the values from any the 3 you just listed.

Reply:
No, it basically only prevents you from mapping the values. (Using these methods they are only ever ints)